features/step_definitions/see_dishes_steps.rb

```diff
@@ -0,0 +1,19 @@
+When(/^I visit the dishes page$/) do
+  visit '/dishes'
+end
+
+Then(/^I should see the page$/) do
+  save_and_open_page
+end
+
+Then(/^I see "([^"]*)"$/) do |arg1|
+  expect(page).to have_content arg1
+end
+
+Then(/^I see the price of "([^"]*)"$/) do |arg1|
```

Comment:
Actually, you should delete these `pending` steps. You aren't using them, so don't leave them in your final code.